minfraud/plugin.spec.yaml

```diff
@@ -6,20 +6,20 @@ title: minFraud
 vendor: rapid7
 support: community
 status: []
-description: The MaxMind minFraud services help identify fraudulent online transactions
+description: Help identify fraudulent online transactions, not just e-commerce; but user account creation, and application installations
 version: 1.0.1
```

Comment:
This needs to be a major version bump since titles changed.

Reply:
Done